tests/api/v1/translations.rs

```
@@ -100,45 +115,57 @@ mod tests {
+        let result: DeletedResult = json::decode(&content).unwrap();
 
-        assert_eq!(StatusCode::NoContent, response.status);
+        assert_eq!(StatusCode::Ok, response.status);
+        assert_eq!(2, result.deleted_translations);
 
         // We fetch all translations
         let (response, content) = get("/api/v1/translations", valid_token());
 
         assert_eq!(StatusCode::Ok, response.status);
 
-        let translations_3 = parse_translations(&content);
+        let translations_3 = parse_translations_by_locales(&content);
 
         assert_eq!(1, translations_3.len());
         assert_eq!(6, translations_3.get(&"ui.add".to_string()).unwrap().len());
-    }
 
-    #[test]
-    fn test_delete_without_key() {
-        let (response, _) = delete("/api/v1/translations", "{}".to_string(), valid_token());
+        // We fetch all translations with key `test.hello`
+        let (response, content) = get("/api/v1/translations/test.hello", valid_token());
 
-        assert_eq!(StatusCode::BadRequest, response.status);
+        assert_eq!(StatusCode::Ok, response.status);
+
+        let translations_4 = parse_translations(&content);
+
+        assert_eq!(2, translations_4.len());
+        assert_eq!("test.hello", translations_4[0].key);
+        assert_eq!("test.hello", translations_4[1].key);
+        assert!(translations_4[0].deleted_at.is_some());
+        assert!(translations_4[1].deleted_at.is_some());
     }
 
-    fn delete_translations(key_to_delete: &'static str) -> (Response, String) {
-        #[derive(RustcEncodable)]
-        struct Body { key: &'static str }
+    #[test]
+    fn test_delete_with_a_key_without_translations() {
+        let (response, content) = delete("/api/v1/translations/not.found.key", valid_token());
+        let result: DeletedResult = json::decode(&content).unwrap();
 
-        let body = json::encode(&Body { key: key_to_delete }).unwrap();
+        assert_eq!(StatusCode::Ok, response.status);
```

Should it be OK ? I would have said that trying to remove an already removed translation should return an error instead (a 404 Not Found or a 410 Gone).